Tests/UnitTests/Scripts/tests.js

```diff
@@ -165,6 +165,20 @@ describe("XMLHTTPRequest", function () {
         expect(xhr).to.have.property('readyState', 4);
         expect(xhr).to.have.property('status', 200);
     });
+
+    if (hostPlatform === "macOS" || hostPlatform === "Unix" || hostPlatform === "Win32") {
+        it("should load URL pointing to symlink", async function () {
+            this.timeout(1000);
```

Comment:
what's this for?

Reply:
I added this a while back to get the test working early on, but maybe it works without it now. Let me check.

Reply:
ya it works without it